examples/kafka/v1beta1/singlescramsecretassociation.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: kafka.aws.upbound.io/v1beta1
+kind: SingleScramSecretAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: example
```

Comment:
Could you give this a more specific name, and then keep it consistent for all the resources?

Reply:
Sure

But what is a difference between examples and examples-generated? Where example for marketplace is taken from? (in case of scramsecretassociation it contains single resource, however examples and examples-generated of scramsecretassociation include dependencies)

Reply:
`examples-generated` is generated by upjet based on the examples in the terraform provider documentation. This is also how upjet knows to automatically generate certain reference fields. Sometimes the examples are incomplete, or contain HCL that the codegen tool can't handle, so they're generally used as a starting place for the handwritten `examples`, which are what actually get published to the upbound marketplace, and also used as manifests in uptest.